### PR TITLE
comment out partner logos

### PR DESCRIPTION
**Pull request by markthom-as**
## 🚀 What's New

Hid partnership logo sections on home and partners pages

Story in Jira: https://emergentworks.atlassian.net/browse/WEB-131?atlOrigin=eyJpIjoiNjY4ZjllNWQyMmYyNGU1ZmExNTJmMmJkMzQ1NjA0ZTEiLCJwIjoiaiJ9

### CODE REVIEW

**Review by ayreal — APPROVED** [review]
@markthom-as Any reason to comment this out? Can we just remove the sections on both pages and also the `PartnersModule` itself if it's something we don't want to use?

**Review by ayreal — COMMENTED** [review]
https://github.com/emergentworks/website/pull/140/files#diff-20d0b56b708aff395fd8fbe74d78677507d6a392acaf378241136e644f0d135dL20 Can't comment on the diff but there's also the "You can learn more about all the organizations we partner with below our featured partner" copy on the partners page we should remove as well

**Comment by markthom-as** [comment]
@ayreal removed the copy and the commented code!

File: src/pages/index.js [hunk 1]
```diff
@@ -182,19 +182,7 @@ const IndexPage = () => {
...
-            heading={<h3>Our Partners</h3>}
-            subheading={
-              <p>
-                We are proud to serve alongside these selected donors, sponsors,
-                and partners:
-              </p>
-            }
-          />
-        </section>
+        </section>x
```
Comment (ayreal) [inline src/pages/index.js:185]:
```suggestion
        </section>
```

Typo here 👁️

**Review by ayreal — APPROVED** [review]

**Pull request merged**